commons-dlp/pom.xml

```diff
@@ -26,5 +26,11 @@
       <artifactId>commons-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.exoplatform.social</groupId>
+        <artifactId>social-component-core</artifactId>
+        <version>6.2.x-ma-SNAPSHOT</version>
```

Comment:
you should not have a version here

Reply:
In addition, commons should not depends on social, as social depends on commons

Reply:
Removed!